operations/research-and-development/product/release-process/release-overview.md

```diff
@@ -43,7 +106,7 @@ Mattermost numbers stable releases in the following format:
 
 **Q: When is release branch cut for a release?**
 
-  - A: One week prior to the release day.
+  - A: Once the Mattermost Cloud release that will be used for the next Self-Managed release has been shipped.
```

Comment:
Hoping we can get more specific on this one to have a more consistent and obvious connection between Cloud and On-premise releases. Not blocking for now, but this is very vague and we need to be more specific in the future.

Reply:
@wiersgallak I changed the text to: "Self-Managed releases are based off of Mattermost Cloud releases. For example, the Self-Managed v5.32.0 release is based off of the ``cloud-2021-01-26`` release tag. Currently the cadence is that the Mattermost Cloud shipped in the last week of a month will become the next Self-Managed release. The release branch for a Self-Managed release will be cut once the Mattermost Cloud release that will be used for the next Self-Managed release has been shipped."

Would something like this be helpful?

Reply:
@amyblais yes, immensely.  Thank you!